tests/test_admin_and_files.py

```diff
@@ -27,7 +27,7 @@ def test_commentaries_registered_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
The use of `assert` is not recommended in Django tests. Instead, use `self.assertTrue(file_exists)` to ensure consistency with Django's testing framework.